pkgs/development/libraries/nlohmann_json/test_data.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenvNoCC
+, lib
+, fetchFromGitHub
+}:
+
+stdenvNoCC.mkDerivation rec {
```

Comment:
Are you sure this is necessary? I was able to just use the plain fetch result, see <https://github.com/NixOS/nixpkgs/pull/134801/commits/c611fa4a51fcfde630b5adc594795ff4773986a3>

Reply:
I was afraid if the fetched data would not have got unpacked.
Glad to know that it works.